ts/src/base/Exchange.ts

```diff
@@ -4264,6 +4264,31 @@ export default class Exchange {
         return [ networkCodeInParams, params ];
     }
 
+    /**
+     * @param {object} params - extra parameters
+     * @param {object} request - existing dictionary of request
+     * @param {string} exchangeSpecificKey - the key for chain id to be set in request
+     * @param {boolean} toUpperCase - either undefined, true or false
+     * @returns {object[]} - returns [request, params] where request is the modified request object and params is the modified params object
+     */
+    handleRequestNetwork (params: any, request: any, exchangeSpecificKey: string, toUpperCase: Bool = undefined) {
```

Comment:
@ttodua we should avoid the `toUpperCase` option specially if it represents three different things while being a bool

Reply:
ok done